src/main/java/in/projecteka/consentmanager/user/PatientsController.java

```diff
@@ -171,4 +174,14 @@ private OtpMediumType getOtpMedium(UserSignUpEnquiry userSignUpEnquiry) {
                         .flatMap(tuple -> signupService.removeOf(tuple.getT2()).thenReturn(tuple.getT1())))
                 : Mono.error(invalidRequester(updateUserRequests.getError()));
     }
+
+    @PostMapping("/change-pin")
+    public Mono<Void> changeTransactionPin(@RequestBody ChangePinRequest request) {
+        return ReactiveSecurityContextHolder.getContext()
+                .map(securityContext -> (Caller) securityContext.getAuthentication().getPrincipal())
+                .flatMap(caller ->
```

Comment:
Can you rename `caller` to something else so that I will be more clear what actually caller is.